playbooks/sample-sap-hypervisor-redhat_ocp_virt-preconfigure.yml

```diff
@@ -4,8 +4,9 @@
   serial: 1
   vars:
     sap_hypervisor_node_platform: redhat_ocp_virt
-
-  tasks:
-    - name: Include Role
-      ansible.builtin.include_role:
-        name: sap_hypervisor_node_preconfigure
+    sap_hypervisor_node_kubeconfig: "{{ lookup( 'ansible.builtin.env', 'KUBECONFIG') }}"
```

Comment:
Could you please elaborate why double assignment needed for KUBECONFIG? Ansible process will start environment variables set for the parent process. So I do not see why this double assignment is required.

Reply:
Actually same as above, some modules / tasks use KUBECONFIG, some K8S_AUTH_KUBECONFIG. Thats why they _are_ set identically. The value is read from the environment from the calling environment and has to be set specifically for ansible.